python/tskit/metadata.py

```diff
@@ -0,0 +1,115 @@
+# MIT License
+#
+# Copyright (c) 2018-2019 Tskit Developers
+# Copyright (c) 2017 University of Oxford
```

Comment:
This started in 2020, so the copyright should be 
```
# Copyright (c) 2020 Tskit Developers
```

Reply:
Fixed https://github.com/tskit-dev/tskit/pull/491/commits/267ff13c7d0481f6f0a5d3d7134553d185395cb8